src/generator.cs

```diff
@@ -2300,7 +2300,10 @@ public void Go ()
 					if (AttributeManager.HasAttribute<CoreImageFilterPropertyAttribute> (pi))
 						continue;
 
-					if (AttributeManager.HasAttribute<WrapAttribute> (pi.GetGetMethod ()) || AttributeManager.HasAttribute<WrapAttribute> (pi.GetSetMethod ()))
+					// we ensure that we do not get a NRE in the case we forgot and ExportAttr in a property
```

Comment:
`in the case we forgot and ExportAttr in a property`
comment is unclear, you mean:
* `an` instead of `and` ?
* `[Wrap] attribute` instead of `ExportAttr` ?

Reply:
The issue happens when we forget an ExportAttr and we do not have a WrapAttr. I'll try to clarify it.

Reply:
maybe
``// Ensure there's a [Wrap] on either (or both) the getter and setter - since we already know there's no [Export]